src/pages/workspaces/workspace/Workflows.js

```diff
@@ -156,102 +156,48 @@ const WorkflowCard = Utils.memoWithName('WorkflowCard', ({ listView, name, names
-          div([synopsis || (selectedWorkflowDetails && 'None')]),
-          div({ style: { fontSize: 18, fontWeight: 600, margin: '1rem 0 0.5rem' } }, ['Method Owner']),
-          div([_.join(',', managers)])
-        ]),
-        div({ style: { margin: '0 1rem', display: 'flex', flexDirection: 'column' } }, [
-          h(ButtonPrimary, { style: { marginBottom: '0.5rem' }, onClick: () => this.exportMethod() }, ['Add to Workspace']),
-          h(ButtonOutline, { onClick: () => this.setState({ selectedWorkflow: undefined, selectedWorkflowDetails: undefined }) }, ['Return to List'])
-        ])
-      ]),
-      div({ style: { fontSize: 18, fontWeight: 600, margin: '1rem 0 0.5rem' } }, ['Documentation']),
-      documentation && h(MarkdownViewer, { style: { maxHeight: 600, overflowY: 'auto' } }, [documentation]),
-      (!selectedWorkflowDetails || exporting) && spinnerOverlay
-    ]
+      setFeaturedList(featuredList)
+      setMethods(methods)
+    }
 
-    const renderList = () => [
-      div({ style: { display: 'flex', flexWrap: 'wrap', overflowY: 'auto', height: 400, paddingTop: 5, paddingLeft: 5 } }, [
-        ...(featuredMethods ?
-          _.map(method => h(MethodCard, { method, onClick: () => this.loadMethod(method) }), featuredMethods) :
-          [centeredSpinner()])
-      ]),
-      div({ style: { fontSize: 18, fontWeight: 600, marginTop: '1rem' } }, ['Find Additional Workflows']),
-      div({ style: { display: 'flex', padding: '0.5rem' } }, [
-        div({ style: { flex: 1, marginRight: 10 } }, [h(DockstoreTile)]),
-        div({ style: { flex: 1, marginLeft: 10 } }, [h(MethodRepoTile)])
-      ])
-    ]
-
-    return h(Modal, {
-      onDismiss,
-      showButtons: false,
-      title: selectedWorkflow ? `Workflow: ${selectedWorkflow.name}` : 'Suggested Workflows',
-      showX: true,
-      width: 900
-    }, Utils.cond(
-      [selectedWorkflow, () => renderDetails()],
-      () => renderList()
-    ))
-  }
+    load()
+  })
 
-  async loadMethod(selectedWorkflow) {
-    const { signal } = this.props
-    const { namespace, name, snapshotId } = selectedWorkflow
 
-    this.setState({ selectedWorkflow })
+  // Helpers
+  const loadMethod = async workflow => {
+    setSelectedWorkflow(workflow)
     try {
-      const selectedWorkflowDetails = await Ajax(signal).Methods.method(namespace, name, snapshotId).get()
-      this.setState({ selectedWorkflowDetails })
+      const { namespace, name, snapshotId } = workflow
+      const details = await Ajax(signal).Methods.method(namespace, name, snapshotId).get()
+      setSelectedWorkflowDetails(details)
     } catch (error) {
       reportError('Error loading workflow', error)
-      this.setState({ selectedWorkflow: undefined })
+      setSelectedWorkflow(undefined)
     }
   }
 
-  async exportMethod() {
-    const { namespace, name, ws } = this.props
-    const { selectedWorkflow } = this.state
-
-    this.setState({ exporting: true })
+  const exportMethod = async () => {
```

Comment:
nit: I see that the additional line returns within this call existed before this PR, but maybe they can be removed (tried to select lines 200 - 212 but github threw an error)?